x/emissions/module/rewards/scores.go

```diff
@@ -305,7 +312,16 @@ func GenerateForecastScores(
 		return nil, err
 	}
 
+	// Calculate forecast task score and emit
+	forecastingTaskScore, err := networkLosses.NaiveValue.Sub(networkLosses.CombinedValue)
```

Comment:
this isn't right... you shouldn't have to recalculate a whitepaper formula -- they should all already be there...